app/assets/javascripts/TT.js

```diff
@@ -196,6 +207,11 @@ var TT = function() {
     }
   }
 
+  var pause = function() {
```

Comment:
I think this function should be called `togglePause`. I found it a little bit strange to see it used inside the block:

```javascript
if('running' == TT.getStatus() || 'paused' == TT.getStatus()) {
  TT.pause();
  //...
```

I thought: "why running `TT.pause()` if `'paused' == TT.getStatus()`?